substrate/frame/utility/README.md

```diff
@@ -22,6 +22,9 @@ This module contains two basic pieces of functionality:
 Since proxy filters are respected in all dispatches of this module, it should never need to be
 filtered by any proxy.
 
+Pre- and post-hooks can be configured via `BatchHook` associated type. 
+They are triggered for each batch call: `batch`, `batch_all`, and `force_batch`. Use the unit type `()` if no behavior is required.
+
```

Comment:
I think the readme is generated from the doc in lib.rs, you can copy the change there.